### ISSUE TITLE
Release 3.1.2 is causing a SyntaxError

### ISSUE THREAD

**Issue by samuelgrigolato** [ymyzk/tox-gh-actions#182]
Syntax Error when running the 3.1.2 version. Here's the relevant console output:

```
> elif isinstance(options.parallel, int) and options.parallel = 0:
SyntaxError: invalid syntax
```

Seems related to this commit: a6451af39

**Comment by ymyzk** [comment]
Hello, thanks for reporting the issue. For some reason, the test failure was overlooked by myself. I merged the PR and released v3.1.3 which should fix the bug.

**Comment by samuelgrigolato** [comment]
Yes, the new patch release solved the issue. Thanks for the quick response :)
